blockpage.js

```diff
@@ -1,189 +1,17 @@
-// Keep service worker alive
```

Comment:
This code is necessary to ensure that the web service renews after becoming inactive so that the extension always works

Reply:
> This code is necessary to ensure that the web service renews after becoming inactive so that the extension always works

@Weredime please ensure this code remains

Reply:
although this may still be working - difficult to test as the service workers don't appear to have an exact timeout lifespan, let me know if this fixes it???

Reply:
> This code is necessary to ensure that the web service renews after becoming inactive so that the extension always works

I'm unsure what "web service" you mean?